### PR TITLE
feat/roll_dice

### PR DESCRIPTION
**Pull request by Eccleria**

### CODE REVIEW

File: src/commands/index.js [hunk 1]
```diff
@@ -61,7 +61,28 @@ const ignoreChannel = {
   admin: true,
 };
 
-const commands = [helloWorld, ignore, reminder, birthday, ignoreChannel];
+const roll = {
+  name: "roll",
+  action: async (message) => {
+    const args = message.content.toLowerCase().split(" ");
+    const diceNumbers = args[1].split("d").map(nb => Number(nb));
+    if (args[1] && diceNumbers[0] && diceNumbers[1]) {
```
Comment (Titch88) [inline src/commands/index.js:69]: you should also check that diceNumbers[0] and diceNumbers[1] are actually numbers (use isNaN(Number(diceNumbers[0]) ,  for example

**Review by Titch88 — APPROVED** [review]

**Comment by Eccleria** [comment]
Add a limit like 20d100 and the prompt of every single results plus the total one.

File: src/commands/index.js [hunk 2]
```diff
@@ -61,7 +61,28 @@ const ignoreChannel = {
...
 
-const commands = [helloWorld, ignore, reminder, birthday, ignoreChannel];
+const roll = {
+  name: "roll",
+  action: async (message) => {
+    const args = message.content.toLowerCase().split(" ");
+    const diceNumbers = args[1].split("d").map(nb => Number(nb));
+    if (args[1] && !isNaN(diceNumbers[0]) && !isNaN(diceNumbers[1])) {
+      let total = 0;
+      for (let i = 0; i < diceNumbers[0]; i++) {
```
Comment (Titch88) [inline src/commands/index.js:71]: WHile we are here, you can also fix this and use a reduce instead of a for .. in

**Comment by Eccleria** [comment]
rebase errors

**Pull request closed without merging**